shared/js/ui/templates/shared/email-feedback-footer.es6.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function (browserInfo, url) {
+  let footer = '---\n'
+  let extensionVersion = chrome.runtime.getManifest().version
```

Comment:
I was considering popping this in a model somewhere, but I'm not sure if it's worth it, since the value is a straightforward string that doesn't change at runtime.

Reply:
Na, it's fine there